src/js/utils/JobUtil.js

```diff
@@ -78,18 +78,27 @@ const JobUtil = {
       Object.assign(spec.run, { docker });
     }
 
+    // default
+    var scheduleId = "default";
```

Comment:
we can use the `let` keyword here instead of var. Var scoping is more complicated to reason about due to hoisting.